docs/source/development/style-guide.md

```diff
@@ -6,6 +6,7 @@ This document serves as a style guide for all `imap_processing` software develop
 the code is committed to the `main` or `dev` branches.
 
 
+(tl;dr)=
```

Comment:
I added these header links because it's a change in MyST. That's the only change to the file, and it functions and looks the same.